### PR TITLE
added properties file

### PR DESCRIPTION
**Pull request by Vosker**

### CODE REVIEW

**Review by mikhail771 — APPROVED** [review]
Good job!

File: src/main/java/spring/intro/config/AppConfig.java [hunk 1]
```diff
@@ -3,25 +3,32 @@
...
 
 @Configuration
+@PropertySource("classpath:db.properties")
 @ComponentScan(basePackages = {
         "spring.intro.service",
         "spring.intro.dao"
 })
 public class AppConfig {
+    @Autowired
+    private Environment environment;
```
Comment (RomanDubovskyi) [inline src/main/java/spring/intro/config/AppConfig.java]: let's user constructor injection

**Comment by RomanDubovskyi** [comment]
travis failed and change PR name from "+properties" to smt more specific

**Comment by RomanDubovskyi** [comment]
PR name wasn't fixed

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**